SamplesDashboard/SamplesDashboard/ClientApp/src/types/samples.ts

```diff
@@ -10,9 +10,12 @@ export interface IRepositoryItem {
     issues: any;
     stargazers: any;
     forks: any;
-    url: string;
+    url: any;
     featureArea: string;
     vulnerabilityAlerts: any;
+    ownerUrl: string;
```

Comment:
>ownerUrl [](start = 4, length = 8)

is this used?

Reply:
Oh yes I forgot to get rid of it. Cleaning up now